readme.md

```diff
@@ -1,7 +1,7 @@
 # Moyo header
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://rimidbit.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https://rimidbit.github.io/layout_moyo-header/report/html_report/)
```

Comment:
please check your links. they lead to 404 page

Reply:
my nick on github rimidbit. I have no idea what`s wrong